run-tests.sh

```diff
@@ -7,14 +7,14 @@ npm run seed
 
 if [ "$LOCAL" == "1" ]; then
     export TEST_PORT=8080
-    npm run dev&
+    # npm run dev&
```

Comment:
1. why did this get commented out? 
2. could we perhaps run our jest tests from here rather than do `&& npm run test-redux` in the `package.json` ?